app/controllers/appointments_controller.rb

```diff
@@ -0,0 +1,33 @@
+class AppointmentsController < ApplicationController
+  def new
+    @holds = Hold.active.where(member: current_user.member)
+    @loans = current_user.member.loans.includes(:item).checked_out
```

Comment:
Hey @jim, these are the loans to be returned, we just wanted to confirm that we're listing them correctly here by calling the `checked_out` scope, does that seem right?

Reply:
That looks right to me!